lib/fontoptim.js

```diff
@@ -87,11 +87,17 @@ FontOptim.prototype = {
 	},
 
 	getFontWeight: function(filename) {
-		return /bold/i.test(filename) ? 600 : 400;
+		if (/-thin/i.test(filename))            return 100;
```

Comment:
There should be one space before each `return` and probably a loop over a table would would look cleaner.